packages/hierarchies-react/README.md

```diff
@@ -242,3 +242,70 @@ function MyTreeComponentInternal({ imodelAccess, imodelKey }: { imodelAccess: IM
   return <TreeRenderer rootNodes={rootNodes} {...state} />;
 }
 ```
+
+## Localization
+
+Localization can be enabled for `TreeRenderer` component and `useTree` and `useUnifiedSelectionTree` hooks by providing an object with localized strings that will be used instead of the default english ones.
```

Comment:
```suggestion
Localization can be enabled for `TreeRenderer` component and `useTree` and `useUnifiedSelectionTree` hooks by providing an object with localized strings that will be used instead of the default English ones.
```